triplea_maps.yaml

```diff
@@ -1827,12 +1827,12 @@
 - mapName: House of Habsburg
   mapCategory: EXPERIMENTAL
   url: https://github.com/triplea-maps/house_of_habsburg/archive/master.zip
-  version: 1
+  version: 1.1
```

Comment:
This version number is not related to the map version, and can only be an Integer.
So please change this to 2